cgi-bin/LJ/CreatePage.pm

```diff
@@ -33,8 +33,9 @@ sub verify_username {
     if ($given_username && !$user) {
         $error = LJ::Lang::ml('error.usernameinvalid');
     }
-    if (length $given_username > 25) {
-        $error = LJ::Lang::ml('error.usernamelong');
+    if (length $given_username > $LJ::USERNAME_MAXLENGTH) {
+        $error = LJ::Lang::ml('error.usernamelong1', 
```

Comment:
Don't forget that the DW code convention preference is for spaces within parentheses like these:
 `( length $given_username > $LJ::USERNAME_MAXLENGTH )`

I realize they weren't there before, but if you're editing a line you should add them. It aids readability.